static/app/plugins/components/pluginIcon.tsx

```diff
@@ -102,6 +102,7 @@ export const ICON_PATHS = {
   rocketchat,
   rookout,
   segment,
+  shortcut: clubhouse,
```

Comment:
Can we rename the file too?

Reply:
That’s what I tried at first but it broke the icon and wouldn’t display. Spent a bunch of time trying to figure out why that’s the case but I was thoroughly stumped.